dio/lib/src/adapters/io_adapter.dart

```diff
@@ -197,7 +200,7 @@ class IOHttpClientAdapter implements HttpClientAdapter {
     if (_defaultHttpClient == null) {
       client.idleTimeout = Duration(seconds: 3);
       if (onHttpClientCreate?.call(client) != null) {
```

Comment:
```suggestion
      if (onHttpClientCreate != null) {
```

Reply:
Yea I want to rebase this after https://github.com/cfug/dio/pull/1810